ocs_ci/ocs/cosbench.py

```diff
@@ -591,7 +591,7 @@ def get_result_csv(self, workload_id, workload_name):
         )
         return f"{self.cosbench_dir}/{archive_file}.csv"
 
-    def cosbench_teardown(self):
+    def cleanup(self):
         """
         Cosbench teardown
```

Comment:
Do change this description of this function.

Reply:
Done